app/src/main/AndroidManifest.xml

```diff
@@ -10,173 +10,192 @@
     <uses-permission android:name="android.permission.WAKE_LOCK" /> <!-- Used to play music when the device is locked -->
     <uses-permission android:name="com.android.launcher.permission.INSTALL_SHORTCUT" /> <!-- Used to add themed icons to the launcher -->
 
-    <permission android:name="${applicationId}.MUSIC_BROADCAST_PERMISSION"
-        android:protectionLevel="signature"/>
+    <permission
+        android:name="${applicationId}.MUSIC_BROADCAST_PERMISSION"
+        android:protectionLevel="signature" />
 
-    <uses-permission android:name="${applicationId}.MUSIC_BROADCAST_PERMISSION"/>
+    <uses-permission android:name="${applicationId}.MUSIC_BROADCAST_PERMISSION" />
 
     <supports-screens
         android:largeScreens="true"
-        android:xlargeScreens="true"
-        android:requiresSmallestWidthDp="320"/>
+        android:requiresSmallestWidthDp="320"
+        android:xlargeScreens="true" />
 
     <application
+        android:name=".JockeyApplication"
         android:allowBackup="true"
-        android:icon="@mipmap/ic_launcher"
         android:fullBackupContent="false"
+        android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
-        android:name=".JockeyApplication"
         android:resizeableActivity="true"
         android:supportsRtl="true"
         tools:ignore="GoogleAppIndexingWarning,UnusedAttribute">
 
         <!-- Library Activity -->
-        <activity android:name=".ui.library.LibraryActivity"
-            android:windowSoftInputMode="adjustPan|adjustNothing"
-            android:theme="@style/SplashTheme"/>
+        <activity
+            android:name=".ui.library.LibraryActivity"
+            android:theme="@style/SplashTheme"
+            android:windowSoftInputMode="adjustPan|adjustNothing" />
 
         <activity-alias
             android:name=".activity.LibraryActivity"
-            android:targetActivity=".ui.library.LibraryActivity"
-            android:icon="@mipmap/ic_launcher">
+            android:icon="@mipmap/ic_launcher"
+            android:targetActivity=".ui.library.LibraryActivity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
 
-            <meta-data android:name="android.app.shortcuts" android:resource="@xml/shortcuts"/>
+            <meta-data
+                android:name="android.app.shortcuts"
+                android:resource="@xml/shortcuts" />
```

Comment:
Thoughts on keeping tags like this as a single-line? I can kinda go either way TBH.